docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1019,6 +1019,29 @@ paths:
       responses:
         '204':
           description: Subscription was deleted
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+    put:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Updates a subscription.
```

Comment:
Do you think it makes sense to describe here which parameters of subscription it is possible to update? I'm afraid users will try to update owning app and list of event-types.